parser.go

```diff
@@ -381,6 +381,19 @@ func (r *Rule) option(key item, l *lexer) error {
 		r.Vars[name] = v
 		lastContent := r.Contents[len(r.Contents)-1]
 		lastContent.Options = append(lastContent.Options, &ContentOption{Name: key.value, Value: strings.Join(parts, ",")})
+	case key.value == "flowbits":
+		nextItem := l.nextItem()
+		parts := strings.Split(nextItem.value, ",")
+		tmp_fb := &Flowbit{}
```

Comment:
this isn't 'tmp' because this is the pointer that you're appending, just call it 'fb' or similar